### ISSUE TITLE
Step-07, Number of client undefined

### ISSUE THREAD

**Issue by jonalbertini** [googlecodelabs/webrtc-web#13]
the client count is broken, please replace:
```javascript
var numClients = io.sockets.sockets.length;
````
by 
```javascript
var numClients = Object.keys(io.sockets.sockets).length;
```

as the #5 already mentioned

**Comment by aaronang** [comment]
@samdutton This issue can be closed since it has been mentioned in issue #5 already.

@jonalbertini It would have been better if you put your proposed solution in the existing issue, or create a pull request.

**Comment by samdutton** [comment]
@jonalbertini — many thanks for the heads-up, and apologies for responding so late.

I'll see if we can get this fixed with the code from #5 as a PR.
